pubsub/awssnssqs/awssnssqs_test.go

```diff
@@ -297,10 +297,13 @@ func TestOpenTopicFromURL(t *testing.T) {
 
 	ctx := context.Background()
 	for _, test := range tests {
-		_, err := pubsub.OpenTopic(ctx, test.URL)
+		top, err := pubsub.OpenTopic(ctx, test.URL)
```

Comment:
Nit: is "top" a short name we use frequently? Seems a bit mysterious, and saves only 2 chars.

Reply:
Done.